src/renderer/entities/notification/ui/NotificationRow/NotificationRow.tsx

```diff
@@ -1,41 +1,37 @@
 import { format } from 'date-fns';
 import { TFunction, Trans } from 'react-i18next';
+import { useUnit } from 'effector-react';
 
-import { BodyText, FootnoteText, Identicon } from '@renderer/shared/ui';
+import { BodyText, FootnoteText } from '@renderer/shared/ui';
 import {
   MultisigAccountInvitedNotification,
   MultisigNotification,
   MultisigNotificationType,
   Notification,
 } from '../../model/notification';
 import { useI18n } from '@renderer/app/providers';
-import { toAddress } from '@renderer/shared/lib/utils';
+import { WalletIcon, walletModel } from '@renderer/entities/wallet';
+import { Wallet } from '@renderer/shared/core';
 
 const NotificationBody = {
-  [MultisigNotificationType.ACCOUNT_INVITED]: (n: Notification, t: TFunction) => {
+  [MultisigNotificationType.ACCOUNT_INVITED]: (n: Notification, t: TFunction, w?: Wallet) => {
+    if (!w) {
+      return <div className="flex"></div>;
```

Comment:
what is the reason for this div? Why not simple fragment?

Reply:
It was like that, but fragment is better so I replaced it with fragment